src/Pages/Result-Page/result.css

```diff
@@ -0,0 +1,136 @@
+/* Rules Page Starts Here*/
+.result-page {
+  font-family: "Montserrat", sans-serif;
+  background-color: var(--primary-color);
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+/* Results Header Start Here */
+.result--header {
+  display: flex;
+  justify-content: center;
```

Comment:
this is correct , if possible you can try this . In both the classes have same 2 properties you can  use result-header classname in above with result-page and remove these 2 properies from result-page

Reply:
Okay, I will update,ThankYou